src/plugins/condition/components/Condition.vue

```diff
@@ -1,34 +1,20 @@
 <template>
-<div id="conditionArea"
+<div v-if="condition"
+     id="conditionArea"
      class="c-cs-ui__conditions"
-     :class="['widget-condition', { 'widget-condition--current': isCurrent }]"
+     :class="['widget-condition', { 'widget-condition--current': currentConditionIdentifier && (currentConditionIdentifier.key === conditionIdentifier.key) }]"
```

Comment:
We should change this to use `openmct.objects.makeKeyString(<identifier>)` eventually.

Reply:
Added TODO